### PR TITLE
[shorturl] update

### PR DESCRIPTION
**Pull request by bw1**
\- https://metamark.net/
\+ self check

@tsaavik

### CODE REVIEW

**Comment by tsaavik** [comment]
Looks good to me!

File: scripts/shorturl.pl [hunk 1]
```diff
@@ -17,14 +17,15 @@
...
+$VERSION = "20210108"; # Fixed and enhanced by tsaavik (dave000@hellspark.com)
 %IRSSI = (
 	authors					=>	"eo, tsaavik",
 	contact					=>	'irssi@eosin.org, dave001@hellspark.com',
 	name						=>	"shorturl.pl",
 	description	   		=>	"Private/Public url reduction script.",
 	license					=>	"GPLv2",
-	changed					=>	"$VERSION"
+	changed					=>	"$VERSION",
+	selfcheckcmd			=> '/shorturl_selfcheck',
```
Comment (ailin-nemui) [inline scripts/shorturl.pl:28]: no /

File: scripts/shorturl.pl [hunk 2]
```diff
@@ -236,10 +244,32 @@ sub goodchan {
...
+	$server->{connected}=1;
+	InjectUrl("https://scripts.irssi.org/", $server, $witem);
+	my $s="ok";
+	if ($last_short_url !~ m/^http/) {
+		$s="Error: no http in shortlink";
+	}
+	$min_url_length=$old_min_url_length;
+	$send_to_channel=$old_send_to_channel;
+	Irssi::print("shorturl: $s");
+	Irssi::command("selfcheckhelperscript $s");
```
Comment (ailin-nemui) [inline scripts/shorturl.pl:265]: add conditional check if selfcheckhelperscript is loaded

**Comment by bw1** [comment]
tab, `\` and check for helper

**Pull request merged**